packages/imperative/CHANGELOG.md

```diff
@@ -2,11 +2,16 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Allows for profile specification on `zowe config secure` command to allow for exclusively prompting for a single profile's secure values. [#1890] https://github.com/zowe/zowe-cli/issues/1890
```

Comment:
This could be a little clearer. Can we change this to: Added the ability to specify a profile with the `zowe config secure` command. This allows the user to prompt for the secure values of the specified profile.